src/coreclr/jit/gentree.cpp

```diff
@@ -24124,6 +24158,51 @@ GenTree* Compiler::gtNewSimdSumNode(var_types type, GenTree* op1, CorInfoType si
 #endif // !TARGET_XARCH && !TARGET_ARM64
 }
 
+#if defined(TARGET_XARCH)
+GenTree* Compiler::gtNewSimdTernaryLogicNode(var_types   type,
```

Comment:
Can you add function header?

Reply:
Yes. Do you mind if I do that in a follow up PR?

We have quite a few of these missing so it'd be nice to get them all at once and avoid the additional CI churn on the main change.

Reply:
> Yes. Do you mind if I do that in a follow up PR?

Oh yes, that's fine.

Reply:
Fixed this one now as I missed a `mustExpand` case for `RotateLeft(Vector###<long>)`